packages/beeq/src/components/date-picker/_storybook/bq-date-picker.stories.tsx

```diff
@@ -0,0 +1,296 @@
+        value=${ifDefined(args.value)}
+        .format-options=${args['format-options']}
+        .isDateDisallowed=${isDate}
+        @bqBlur=${args.bqBlur}
+        @bqChange=${args.bqChange}
+        @bqClear=${args.bqClear}
+        @bqFocus=${args.bqFocus}
+      >
+        ${!args.noLabel ? label : nothing}
+        ${args.prefix ? html`<bq-icon name="user-circle" slot="prefix"></bq-icon>` : nothing}
+        ${args.suffix ? html`<bq-icon name="arrow-down" slot="suffix"></bq-icon>` : nothing}
+      </bq-date-picker>
+    </div>
+  `;
+};
+
+export const InitialValue: Story = {
+  render: (args) => html`
+    <div class="flex flex-row gap-4">
+      <!-- Default date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Default date picker</p>
+        ${Template({ ...args, value: '2024-05-25', name: 'bq-date-picker-default', noLabel: 'true' })}
+      </div>
+      <!-- Range date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Range date picker</p>
+        ${Template({
+          ...args,
+          value: '2024-06-04/2024-06-15',
+          name: 'bq-date-picker-range',
+          type: 'range',
+          months: 2,
+          noLabel: 'true',
+        })}
+      </div>
+      <!-- Multi date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Multi date picker</p>
+        ${Template({
+          ...args,
+          value: '2024-05-08 2024-05-09 2024-05-10',
+          name: 'bq-date-picker-multi',
+          type: 'multi',
+          months: 2,
+          noLabel: 'true',
+        })}
+      </div>
+    </div>
+  `,
+};
+
+export const MixMax: Story = {
+  name: 'Min and Max allowed dates',
+  render: Template,
+  args: {
+    min: '2024-06-05',
+    max: '2024-06-15',
+  },
+};
+
+export const Disabled: Story = {
+  render: Template,
+  args: {
+    disabled: true,
+  },
+};
+
+export const ValidationStatus: Story = {
+  name: 'Validation',
+  render: (args) => html`
+    <div class="flex flex-row gap-4">
+      <!-- Error -->
+      <div class="flex flex-col gap-2">
+        <p>Error date picker</p>
+        ${Template({ ...args, name: 'bq-date-picker-error', 'validation-status': 'error', noLabel: 'true' })}
+      </div>
+      <!-- Success -->
+      <div class="flex flex-col gap-2">
+        <p>Success date picker</p>
+        ${Template({ ...args, name: 'bq-date-picker-success', 'validation-status': 'success', noLabel: 'true' })}
+      </div>
+      <!-- Warning -->
+      <div class="flex flex-col gap-2">
+        <p>Warning date picker</p>
+        ${Template({ ...args, name: 'bq-date-picker-warning', 'validation-status': 'warning', noLabel: 'true' })}
+      </div>
+    </div>
+  `,
+};
+
+export const Optional: Story = {
+  name: 'Label with "Optional"',
+  render: Template,
+  args: {
+    optionalLabel: true,
+  },
+};
+
+export const Tooltip: Story = {
+  name: 'Label with "Info tooltip"',
+  render: Template,
+  args: {
+    hasLabelTooltip: true,
+  },
+};
```

Comment:
For this specific example, I would recommend to add also:

```js
  parameters: {
    layout: 'centered',
  },
```

to the Tooltip story object.